packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -281,11 +291,11 @@ const createFlowsheetResultCellDisplay = (resultDataSet, hideUnit, numericOverfl
     additionalResultList.forEach((result) => {
       const { isSingleResult, isPairedResult } = checkIfSingleOrPairedResult(result);
       if (isSingleResult) {
-        const resultInterpretation = !isEmpty(result.interpretation) && !result.isUnverified ? result.interpretation : null;
+        const resultInterpretation = !isEmpty(result.interpretation) && !result.isUnverified ? result.interpretation.trim().toLowerCase() : null;
```

Comment:
https://github.com/cerner/terra-clinical/pull/574#discussion_r386189233

petition to replace this isEmpty usage anywhere in the component with the double bang

Reply:
Can't -- Linter doesn't like it how we have it currently written as the comparison side of the ternaries.  https://eslint.org/docs/rules/no-extra-boolean-cast. (It's fine to double cast when setting/returning values to new variables, but not fine to double-cast for bool comparisons.)

+1 to the Suggestion and https://github.com/cerner/terra-clinical/pull/574#discussion_r386189233 - better option we've talked about is probably refactor to remove these entirely -- one idea is an immediate result object and  initializer function to detect all of  the prop object and string checks like these up front, and then we can just utilize straight booleans downstream throughout the rest of the project.

Reply:
i'm confused, would a simple existence check suffice then?
```js
const resultInterpretation = result.interpretation && !result.isUnverified ? result.interpretation.trim().toLowerCase() : null;
```